src/db-up/azext_db_up/_params.py

```diff
@@ -58,6 +59,20 @@ def load_arguments(self, _):  # pylint: disable=too-many-locals, too-many-statem
         c.argument('version', help='Server version', default='5.7')
 
     with self.argument_context('postgres up') as c:
+        c.argument('server_name', options_list=['--server-name', '-s'], help='Name of the server.',
+                   local_context_attribute=LocalContextAttribute(
+                       name='postgres_server_name', actions=[LocalContextAction.SET], scopes=['cupertino']
```

Comment:
what's this scope for?

Reply:
`cupertino` is a standalone CLI extension, you can find it [here](https://github.com/VSChina/azure-connect). It is a client of azure connect service, which required for local context support.

Reply:
got it. While just thinking how to best describe it, since cupertino should just be a project name.

Reply:
Actually it is also the root command group name. The scope of local context must be defined with a valid command group or command name.